modules/heatpump.py

```diff
@@ -32,7 +32,7 @@ def __check_outside_temperature_range__(self, temp_celsius: float) -> bool:
         Returns:
             bool: True if in range
         """
-        return temp_celsius > -100 and temp_celsius < 100
+        return temp_celsius > -100 and temp_celsius < 100  # noqa: PLR2004
```

Comment:
There will always going to be a lot of magic values in this project. At this point it would complicate things to ask developers to put magic comments at them.

It's more important to get configuration and architecture right.
Automated checks saying "don't do this" without good architecture for how to do it better may create a mess.

Reply:
removed the rules